docs/self-managed/identity/what-is-identity.md

```diff
@@ -0,0 +1,15 @@
+---
+id: what-is-identity
+title: "What is Identity?"
+sidebar_label: "What is Identity?"
+---
+
+Identity is the component within the Camunda Cloud stack responsible for authentication and authorization. It allows you to manage:
+- Applications
+- APIs
+- Permissions
+- Roles
+
+### That's great, but what do I do now?
+
+If you're new to Identity, we suggest reviewing [Step 1: Setup environment](../getting-started/docker/setup-environment/) to get started on your journey!
```

Comment:
😄 I love the voice here, but @christinaausley is going to come through and modify this to meet our style guide.